recipes/pydataverse/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pydataverse" %}
```

Comment:
Please forgive my ignorance but is all lower case preferred in Conda? I only ask because I've usually seen this package as camel case: pyDataverse.

Reply:
Note that, in PyPI, the package name is case insensitive. In conda the lower case form is preferred but that  doesn't affect imports as shown above.